internal/services/datafactory/data_factory_dataset_json_resource.go

```diff
@@ -114,19 +114,17 @@ func resourceDataFactoryDatasetJSON() *pluginsdk.Resource {
 							Default:  false,
 						},
 						"path": {
-							Type:         pluginsdk.TypeString,
-							Required:     true,
-							ValidateFunc: validation.StringIsNotEmpty,
+							Type:     pluginsdk.TypeString,
+							Required: true,
```

Comment:
Would it make more sense for these to be Optional? Or do you think it's best if you specify that it must be empty?

Reply:
Hi, I think maybe it's better to make it `Required` since the empty string is a corner case, in most scenarios the users need to specify a file name and path, making them `Optional` might cause using an empty string unintentionally.

Reply:
That makes sense to me! Thanks for confirming!